draco/analysis/mapmaker.py

```diff
@@ -269,10 +291,23 @@ def _solve_m(self, m, f, v, Ni):
         # Construct pre-wightened beam and beam-conjugated matrices
         bmt = bm * Nh[:, np.newaxis]
         bth = bmt.T.conj()
-
         # Pre-wighten the visibilities
         vt = Nh * v
 
+        a_wiener = self._minimize(m, bmt, bth, vt)
+
+        # Copy the solution into a correctly shaped array output
+        a = np.zeros((bt.telescope.num_pol_sky, bt.telescope.lmax + 1), dtype=v.dtype)
+        a[:, m:] = a_wiener.reshape(bt.telescope.num_pol_sky, -1)
+
+        return a
+
+    def _minimize(self, m, beam, beam_conj, vec):
```

Comment:
Why is this called minimize? The name doesn't seem obvious to me.

Reply:
maybe not the best name ... wanted to indicate that in this routine you are solving a linear matrix equation